search-provider/eks-discovery-feed-provider.c

```diff
@@ -283,6 +313,19 @@ add_key_value_pair_from_model_to_variant (EkncContentObjectModel *model,
   add_key_value_pair_to_variant (builder, underscore_key, value);
 }
 
+static void
+add_key_int_value_pair_from_model_to_variant (EkncContentObjectModel *model,
```

Comment:
The name of this method is a bit odd because it does not take the int. Perhaps the code could be reworked to do so, or the name of the function could be changed.